kuma/wiki/views/edit.py

```diff
@@ -168,16 +168,15 @@ def edit(request, document_slug, document_locale, revision_id=None):
                     # Get the possibly new slug for the imminent redirection:
                     doc = doc_form.save(parent=None)
 
-                    if is_iframe_target:
-                        # TODO: Does this really need to be a template? Just
-                        # shoehorning data into a single HTML element.
-                        response = HttpResponse(textwrap.dedent("""
-                            <span id="iframe-response"
-                                  data-status="OK"
-                                  data-current-revision="%s">OK</span>
-                        """ % doc.current_revision.id))
-                        response['X-Frame-Options'] = 'SAMEORIGIN'
-                        return response
+                    # save-and-edit button
+                    if is_async_submit:
+                        # This is the most recent revision id
```

Comment:
This branch (valid AJAX edit to the document) is uncovered by tests. I'm also not sure if it is possible to get here (is AJAX allowed for doc-only saves?)